packages/mock-account-provider/app/routes/webhooks.ts

```diff
@@ -96,11 +96,17 @@ export async function handleOutgoingPaymentCreated(wh: WebHook) {
   const acc = await mockAccounts.getByPaymentPointer(pp)
 
   if (!acc) {
-    throw json('No account found for payment pointer', { status: 500 })
+    throw json('No account found for payment pointer', { status: 200 })
   }
 
   const amt = parseAmount(payment['sendAmount'])
-  await mockAccounts.pendingDebit(acc.id, amt.value)
+
+  try {
+    await mockAccounts.pendingDebit(acc.id, amt.value)
+  } catch (e) {
+    const errorInfo = e instanceof Error && e.stack ? e.stack : String(e)
+    throw json(errorInfo, { status: 200 })
```

Comment:
Another option is for us to just reply with `200` after successfully parsing webhook events, instead of throwing `200` on subsequent errors.

See:
https://stripe.com/docs/webhooks#acknowledge-events-immediately